Code/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-
+createExportTestHeaders()
```

Comment:
This morning I figured out that it's not just my imagination I'm waiting a lot longer for `make` to run since this PR was merged. 

This regenerates `RDBoost/export.h` every time cmake runs. Since `RDBoost/export.h` is included in essentially every header file, this means that running `cmake` is basically equivalent to running `make clean` (except it doesn't actually clean anything): a full rebuild is required.

It would be great if we could do something about that.

Reply:
@greglandrum A-ha, that's not good. I am going to fix this later today. Thanks for spotting that!

Reply:
@greglandrum Also, I haven't forgotten about the script to auto-generate `RDKIT_XXX_EXPORT` decorators - hopefully I'll manage to find some time over the weekend.

Reply:
@greglandrum PR #1901 addresses thi issue.